rabbitmq-operator/src/main/java/com/indeed/operators/rabbitmq/reconciliation/rabbitmq/RabbitMQClusterFactory.java

```diff
@@ -195,7 +195,7 @@ private PodDisruptionBudget buildPodDisruptionBudget(final RabbitMQCustomResourc
         return resource.getSpec().getClusterSpec().getUsers().stream()
                 .map(user -> new RabbitMQUser(
                         user.getUsername(),
-                        rabbitMQSecrets.createUserSecret(user.getUsername(), resource),
+                        rabbitMQSecrets.createUserSecret(user.getUsername(), resource), // todo: retrieve the secret if the user already exists
```

Comment:
Are we going to continue trying to open issues for all TODOs left in the code for stuff on Github?

Reply:
probably on a case-by-case basis. the todo here is completely unrelated to this change and the code was already there, just figured i would mark it. i did open an issue for this particular item.